examples/integrations/gcp/src/jobs/etl/text/classification/classification_etl.py

```diff
@@ -25,12 +25,20 @@
 
 VERTEX_MIN_TRAINING_EXAMPLES = 1
 
+Partition = Union[Literal['training'], Literal['test'], Literal['validation']]
+# Maps from our partition naming convention to google's
+partition_mapping = {
+    'training': 'train',
+    'test': 'test',
+    'validation': 'validation'
+}
+
 # Optionally set env var for testing
 _labelbox_api_key = os.environ.get('LABELBOX_API_KEY')
 if _labelbox_api_key is None:
     client = secretmanager.SecretManagerServiceClient()
     secret_id = "labelbox_api_key"
-    name = f"projects/{os.environ['GOOGLE_PROJECT']}/secrets/{secret_id}/versions/1"
+    name = f"projects/{os.environ['GOOGLE_PROJECT']}/secrets/{secret_id}/versions/2"
```

Comment:
Can you please revert this back? Otherwise this won't be compatible with the deployment script

Reply:
yep, sorry forgot it